src/MuTalk-Model/MTAnalysis.class.st

```diff
@@ -636,6 +640,14 @@ MTAnalysis >> operators: anObject [
 	operators := anObject
 ]
 
+{ #category : 'testing' }
+MTAnalysis >> resultsContainUnexpectedFailuresOrErrorsFor: results [
+
+	^ results anySatisfy: [ :result |
+		  result unexpectedFailureCount > 0 or: [
```

Comment:
All the logic of the mutation framework should not use `failures` instead of `unexpectedFailureCount`, that take also `unexpectedPasses` into account?

In the sense that changes in a `expectedFailure` test can make it pass and blah...